src/StackExchange.Redis/ExceptionFactory.cs

```diff
@@ -312,7 +312,7 @@ ServerEndPoint server
             // Add server data, if we have it
             if (server != null && message != null)
             {
-                var bs = server.GetBridgeStatus(message.Command);
+                var bs = server.GetBridgeStatus(message.IsForSubscriptionBridge ? ConnectionType.Subscription: ConnectionType.Interactive);
```

Comment:
Turns out we were also getting the wrong error stats, for the command we sent to the wrong place - fixing that.